var/spack/repos/builtin/packages/git/package.py

```diff
@@ -56,7 +55,7 @@ class Git(Package):
     # Also depends_on gettext: apt-get install gettext (Ubuntu)
```

Comment:
I think we should wait and do a separate little PR when/if #1156 lands to add ] depends_on('gettext') here. 

Agreed?